Mesh_3/include/CGAL/Mesh_3/experimental/Facet_topological_criterion_with_adjacency.h

```diff
@@ -109,7 +113,11 @@ public Mesh_3::Abstract_criterion<Tr, Visitor_>
         break;
       }
     }
-
+    if(nb_vertices_on_curves == 3) {
+      return Badness(Quality(1)); // bad!
```

Comment:
TODO: We should verify if all three vertices are a common set of patches with more than 1 patch, and return "bad" only in that case. If the three vertices are on only one patch, then the triangle should not be tagged as "bad".